### PR TITLE
Master tmp2

### PR DESCRIPTION
**Pull request by jiyong2**
### Change Description ###

Add guide for media controller operation

### API Changes ###

 - ACR: ACR-1241

### CODE REVIEW

File: docs/application/native/guides/app-management/common-appcontrols.md [hunk 1]
```diff
@@ -1809,6 +1809,70 @@ retrieve_media_operation(const char* mime_type)
 }
 ```
 
+### Creates media controller to communicate with application
+
+The media controller operation is supported in mobile and wearable applications.
+To control media application, use the `APP_CONTROL_OPERATION_MEDIA_CONTROLLER` operation with the `server` and `client` launch type.
+To specify various details refer to the extras defined below.
```
Comment (sabeena21) [inline docs/application/native/guides/app-management/common-appcontrols.md]: Global: 
Please rephrase as following:
To specify various event details, see Extra Input and Extra Output.

Please change this in the entire file.

File: docs/application/native/guides/app-management/common-appcontrols.md [hunk 2]
```diff
@@ -1809,6 +1809,70 @@ retrieve_media_operation(const char* mime_type)
...
+
+- `file:<path>`
+- `http:<path>`
+- `https:<path>`
+
+#### Extra Input
+
+| Key                               | Description                              | Note                  |
+|-----------------------------------|------------------------------------------|-----------------------|
+| `APP_CONTROL_DATA_TYPE` | The launch type of the media controller operation. The available values are 'server' and 'client'. This key must be passed as a string. | This key is mandatory. |
```
Comment (sabeena21) [inline docs/application/native/guides/app-management/common-appcontrols.md]: Please use code tag for `server` and `client`

File: docs/application/native/guides/app-management/common-appcontrols.md (hunk 1 above)
Comment (jiyong2) [reply docs/application/native/guides/app-management/common-appcontrols.md]: The extras include operation, MIME Type, URI, Extra Input, Extra Output and so on. The extras are different for each event. Could you pelease rephrase the sentense "see Extra Input and Extra Output"?
And the people responsible for managing each event and document are different. So I can't change this in the entire file.

**Comment by jiyong2** [comment]
I have updated the documentation. Please review this again.

**Comment by mijin85cho** [comment]
Thanks for the review & update.

**Comment by mijin80lee** [comment]
TD Sync Completed.

**Pull request merged**